### DISCUSSION TITLE
Calculate option view is missing in Budibase version 2.32.4 !

### DISCUSSION THREAD

**Discussion by mallipatil** [Budibase/budibase#14831] [Help] [answered]
I am not sure whether it's defect or setting issue hence asking for help

Document at link https://docs.budibase.com/docs/views suggests option to calculate for view (refer snapshot)
![image](https://github.com/user-attachments/assets/1ccca560-17bf-4ff7-9e21-57506a1c7939)

However I have self-hosted 2.32.4 version - refer snapshot, not able to see "calculate" optn. 
![image](https://github.com/user-attachments/assets/4f87a24b-3a6b-4aec-94da-3b0b904be340)

**Comment by mike12345567** [comment, marked as answer]
Hi @mallipatil - sorry there are still some videos in our documentation which need updated - this feature was removed a while ago from Budibase due to performance issues and the in-ability to support it outside of internal tables. 

However, in the next few weeks we'll be releasing Budibase 3.0 - which will include a replacement for this feature that works across SQL and internal tables.

**Reply by mallipatil** [reply]
Thanks for response.  Great news, will look forward for Budibase 3.0 version

**Reply by EBuddi** [reply]
Hi @mike12345567 

I see from the Budibase website that the update has been made (https://docs.budibase.com/changelog), but I still don't have the configure calculations option on my view table between the Access: App admin and Filter buttons (screenshot below). Do you know if the update is delayed? I just joined Budibase yesterday - is that something to do with it?

<img width="322" alt="Screenshot 2024-11-28 095437" src="https://github.com/user-attachments/assets/7ff8c523-c921-43ed-b41b-072fb3c0964d">

Thanks.

**Reply by mike12345567** [reply]
> Hi @mike12345567 
> 
> I see from the Budibase website that the update has been made (https://docs.budibase.com/changelog), but I still don't have the configure calculations option on my view table between the Access: App admin and Filter buttons (screenshot below). Do you know if the update is delayed? I just joined Budibase yesterday - is that something to do with it?
> 
> <img width="322" alt="Screenshot 2024-11-28 095437" src="https://github.com/user-attachments/assets/7ff8c523-c921-43ed-b41b-072fb3c0964d">
> 
> Thanks.
> 
> 

Hi @EBuddi views that can calculate are now their own independent thing, when you create a new view it will ask you if you wish to create a calculation view. More information can be found here: https://docs.budibase.com/docs/views

**Reply by EBuddi** [reply]
Ah yes,  I see now - thanks very much!
